enable/savage/compliance/crosshair.py

```diff
@@ -84,8 +84,8 @@ def normal_mouse_move(self, event):
         self.svg_coords = event.x, y
         event.handled = True
 
-    @on_trait_change("svg_coords,mouse_in")
-    def ensure_redraw(self):
+    @observe("svg_coords,mouse_in")
+    def ensure_redraw(self, event=None):
```

Comment:
This method and `cell_properties_changed` in a later file are never called standalone, but they are not privately named.  So I made `event` a kwarg to protect against breaks from other callers, but this is likely overkill/unneeded. I am happy to change this to just `event` if a reviewer prefers.